manifest.json

```diff
@@ -92,10 +92,13 @@
             "example": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/codecommit/2015-04-13/examples-1.json",
             "api-reference": "https://docs.aws.amazon.com/codecommit/latest/APIReference",
             "methods": [
+                "CreateRepository",
```

Comment:
could you add an entry in the Changelog of `CodeCommit` for these new operations?

Reply:
@jderusse done 👍